lib/shopify_app/scripttags_manager.rb

```diff
@@ -2,12 +2,8 @@ module ShopifyApp
   class ScripttagsManager
     class CreationFailed < StandardError; end
 
-    def self.queue(shop_name, token)
-      ShopifyApp::ScripttagsManagerJob.perform_later(shop_name: shop_name, token: token)
-    end
-
-    def initialize(shop_name, token)
-      @shop_name, @token = shop_name, token
+    def self.queue(shop_domain)
+      ShopifyApp::ScripttagsManagerJob.perform_later(shop_domain: shop_domain)
     end
```

Comment:
Fair enough, I was assuming a shop object, but you're right, it might not always be the case. Will fix.